torch/_dynamo/compiled_autograd.py

```diff
@@ -30,6 +30,15 @@ def maybe_clone(x):
     return x
 
 
+def keep_graph(graph, keep_graph):
+    graph.keep_graph = keep_graph
+    return graph
+
+
+def should_keep_graph(graph):
+    return graph.keep_graph
```

Comment:
Why does this need to be in python?

Reply:
Setting breakpoints / debugging is easier ;)

I can move entirely to C++ setting